lightning/src/util/scid_utils.rs

```diff
@@ -32,6 +32,11 @@ pub fn block_from_scid(short_channel_id: &u64) -> u32 {
 	return (short_channel_id >> 40) as u32;
 }
 
+/// Extracts the tx index (bytes [2..4]) from the `short_channel_id`
+pub fn tx_index_from_scid(short_channel_id: &u64) -> u32 {
+	return (short_channel_id >> 16 & MAX_SCID_TX_INDEX) as u32;
```

Comment:
nit: I believe you this is right, but the fact that I have to google "Rust operator precedence" to review it is annoying, can you add extra ()s around the >>?